docs/Map.jhm

```diff
@@ -1 +1 @@
pID target="Breaking Changes" url="breaking-changes.html" default="no"/><mapID target="Deep Dive into DAO" url="deep-dive-into-dao.html" default="no"/><mapID target="Migration-Guide.md" url="migration-guide.html" default="no"/><mapID target="Exposed Modules" url="exposed-modules.html" default="no"/><mapID target="Data Types" url="data-types.html" default="no"/><mapID target="Transactions" url="transactions.html" default="no"/><mapID target="Data+Types" url="data-types.html" default="no"/><mapID target="Samples" url="samples.html" default="no"/><mapID target="Exposed-Modules.md" url="exposed-modules.html" default="no"/><mapID target="Database-and-DataSource.md" url="database-and-datasource.html" default="no"/><mapID target="Exposed+Documentation" url="home.html" default="no"/><mapID target="Migration-Guide" url="migration-guide.html" default="no"/><mapID target="Exposed-Modules" url="exposed-modules.html" default="no"/><mapID target="Data-Types" url="data-types.html" default="no"/><mapID target="Frequently-Asked-Questions" url="frequently-asked-questions.html" default="no"/><mapID target="Getting Started with Exposed" url="getting-started-with-exposed.html" default="no"/><mapID target="Deep+Dive+into+DSL" url="deep-dive-into-dsl.html" default="no"/><mapID target="Contributing" url="contributing.html" default="no"/><mapID target="Exposed+Modules" url="exposed-modules.html" default="no"/><mapID target="Contributing to Exposed" url="contributing.html" default="no"/><mapID target="Contributing.md" url="contributing.html" default="no"/><mapID target="Database-and-DataSource" url="database-and-datasource.html" default="no"/><mapID target="Deep-Dive-into-DAO" url="deep-dive-into-dao.html" default="no"/><mapID target="Transactions.md" url="transactions.html" default="no"/><mapID target="Deep Dive into DSL" url="deep-dive-into-dsl.html" default="no"/><mapID target="Samples.md" url="samples.html" default="no"/><mapID target="Getting-Started-with-Exposed.md" url="getting-started-with-exposed.html" default="no"/><mapID target="Working+with+Database+and+DataSource" url="database-and-datasource.html" default="no"/><mapID target="Home" url="home.html" default="yes"/><mapID target="SQL Functions" url="sql-functions.html" default="no"/><mapID target="Data-Types.md" url="data-types.html" default="no"/><mapID target="Migration+Guide" url="migration-guide.html" default="no"/><mapID target="Working with Database and DataSource" url="database-and-datasource.html" default="no"/><mapID target="Contributing+to+Exposed" url="contributing.html" default="no"/><mapID target="SQL-Functions.md" url="sql-functions.html" default="no"/><mapID target="Getting-Started-with-Exposed" url="getting-started-with-exposed.html" default="no"/><mapID target="-of4r3s_1_json" url="starting-page-Home.json" default="no"/><mapID target="Frequently-Asked-Questions.md" url="frequently-asked-questions.html" default="no"/><mapID target="Migration Guide" url="migration-guide.html" default="no"/></map>
```

Comment:
Is it intended? I expect it would be regenerated after the next CI run.

The whole directory `docs` is regenerated on every release (except files `logo-dark.png`, `logo-light.png`, `site.webmanifest` inside)

Reply:
It looks like it was automatically updated with the conversion of `Getting-Started-with-Exposed.md` to a `.topic` file. Should I exclude this and any changes of the `docs` folder?

Reply:
Yep, files from generated folders will be dropped and regenerated anyway. I would exclude them from the PR to make it easy to read